user-data/variables.tf

```diff
@@ -13,3 +13,19 @@ variable "teleport_server" {
 variable "teleport_auth_token" {
   default = ""
 }
+
+variable "efs_mount_point" {
```

Comment:
Do you still need efs? Now that you fetch the config from S3?

Reply:
yes for the data

Reply:
hmmm but you don't create the efs in this module then? Also, the `efs_id` variable is not used